src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -116,23 +117,70 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 		}
 	}
 
-	public static transformData(rows: string[][], columns: any[]): { [key: string]: string }[] {
+	private static createIconColumn<T extends Slick.SlickData>(col: azdata.TableColumn): Slick.Column<T> {
+		return <Slick.Column<T>>{
```

Comment:
Do you need the cast here? You already define the return type

Reply:
Yes. I need to specify additional property value for headerFilter plugin.